docs/source/data-types/timeuuid.md

```diff
@@ -27,4 +27,47 @@ if let Some(rows) = session.query("SELECT a FROM keyspace.table", &[]).await?.ro
 }
 # Ok(())
 # }
-```
\ No newline at end of file
+```
+
+## Creating your own Timeuuid
+
+To create your own `Timeuuid` objects from timstamp-base `uuid` v1, you need to enable the feature `v1` of `uuid` crate using:
```

Comment:
`base` -> `based`

Reply:
there was also the "timstamp" lol

fixed.